types/file-saver-es/test/file-saver-es-tests.umd.ts

```diff
@@ -0,0 +1,66 @@
+/**
```

Comment:
Is `umd` testing for `es` needed?

Reply:
Yes, this package still exposes a UMD bundle.